test/framework/kogitomanagementconsole.go

```diff
@@ -0,0 +1,100 @@
+// Copyright 2020 Red Hat, Inc. and/or its affiliates
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package framework
+
+import (
+	"fmt"
+
+	"github.com/kiegroup/kogito-cloud-operator/pkg/apis/app/v1alpha1"
+	"github.com/kiegroup/kogito-cloud-operator/pkg/client/kubernetes"
+	"github.com/kiegroup/kogito-cloud-operator/pkg/framework"
+	"github.com/kiegroup/kogito-cloud-operator/pkg/infrastructure"
+	apps "k8s.io/api/apps/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// InstallKogitoManagementConsole deploy the Kogito Management Console
+func InstallKogitoManagementConsole(namespace string, installerType InstallerType, replicas int) error {
```

Comment:
This looks like a copy and paste from other services:
https://github.com/kiegroup/kogito-cloud-operator/blob/master/test/framework/kogitodataindex.go#L32

I think you guys could have a way to simplify this, like having a "Generic service deployment" handler.

Reply:
We could indeed as you merged the whole into a Service Definition deployment on Operator's side.
Created https://issues.redhat.com/browse/KOGITO-1628 as it is not priority yet for DP. We will implement that one for 0.10.0